src/redux/action-types/transactions.types.ts

```diff
@@ -0,0 +1,3 @@
+export const transactionTypes = {
+  GET_ALL_TRANSACTIONA: 'GET_ALL_TRANSACTIONA',
```

Comment:
@w3bh4ck Should this be `Get_All_Transactions` instead

Reply:
@tomijaga yea, that was a typo... I'll correct that now